[Docs] Document behaviour of disposition undeliverable-here=true for users of the standard address space (#5271)
EnMasseProject/enmasse: documentation/modules/con-standard-address-space.adoc

* No selectors on queues
* No browsing on queues
* No message groups

> * Restrictions affecting the use of some AMQP delivery states, for example, the `undeliverable-here=true` modified delivery state when there are competing consumers attached to the same message route.

Reviewer: ```suggestion
* Restrictions affecting the use of some AMQP delivery states, such as the `undeliverable-here=true` modified delivery state when there are competing consumers attached to the same message route.
```
